code/drasil-gen/package.yaml

```diff
@@ -22,6 +23,8 @@ dependencies:
 - drasil-lang
 - drasil-printers
 - drasil-sysinfo
+- drasil-theory
```

Comment:
Good catch! I didn't like that I had to make this dependency explicit. When I added type checking initially, I started with only the EquationalModel InstanceModels and needed to use the function for extracting QDefinitions from InstanceModels that drasil-theory exposes which is why the dependency was added.

However, then I realized there's no reason why I couldn't (nor shouldn't) extend type checking to non-EquationalModels too. So, I created the "TypeChecks" typeclass (perhaps a bad name, I'll try to think of something else), so that we can add type checking to any chunk that contains anything that needs type checking.  Then I instantiated this typeclass for all InstanceModels via ModelKinds. With it, we added type checking to ODEs and everything else too. Notably, "TypeChecks"s "typeCheckExpr" returns a _list_ of expressions and expected types because ODEs expose more than 1 expression to type check (e.g., @cd155's DifferentialModel exposes a whole system of them to type check).

This is a very long explanation to say: I left it there by accident. I will remove it now.